Caboodle/PhoneDialer/PhoneDialer.ios.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Foundation;
+using UIKit;
+using CoreTelephony;
+
+namespace Microsoft.Caboodle
+{
+	public static partial class PhoneDialer
+	{
+		public static bool IsSupported
+		{
+			get
+			{
+				var nsUrl = CreateNsUrl("0000000000");
+				var canCall = UIApplication.SharedApplication.CanOpenUrl(nsUrl);
+				
+				if (canCall)
+				{
+					using (var netInfo = new CTTelephonyNetworkInfo())
+					{
+						var mnc = netInfo.SubscriberCellularProvider?.MobileNetworkCode;
+						return !string.IsNullOrEmpty(mnc) && mnc != "65535"; // 65535 stands for NoNetwordProvider
+					}
+				}
+				
+				return false;
+			}
+		}
+
+		public static void Open(string number)
+		{
+			if (string.IsNullOrWhiteSpace(number))
+			{
```

Comment:
@mattleibow are we standardizing on need or not needing { around a single line if?

Reply:
@jamesmontemagno I prefer always using braces for the simple reason that it avoids confusion due to the grouping. Also it makes changes easier - if we ever have to add another line, then we don't have to also add braces. But, this is a very opinionated thing. 

If we are following the .net foundation guides, then they say that we don't need it. So I guess we don't need them.
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
Then again, we don't really follow most of them.